tests/test_magpie_api.py

```diff
@@ -107,6 +111,155 @@ def setUpClass(cls):
         cls.check_requirements()
         cls.setup_test_values()
 
+    def test_Webhook_CreateUser(self):
```

Comment:
I think we should move the webhook tests into a separate file similar to `test_services` to keep this use case more concise and better isolate specific webhook features. 
(sorry, I know I guided you here in the first place)